bessctl/commands.py

```diff
@@ -1863,15 +1864,15 @@ def print_delta(tc, delta, name_len):
         if not tcs:
             raise cli.CommandError('No traffic class to monitor')
 
-    cli.fout.write('Monitoring traffic classes: %s\n' % ', '.join(tcs))
+    cli.fout.write('Monitoring traffic classes: {}\n'.format(', '.join(tcs)))
 
     last = {}
     now = {}
 
     for tc in tcs:
         last[tc] = cli.bess.get_tc_stats(tc)
 
-    try:
+    def print_loop(csv=None):
         while True:
```

Comment:
How about moving this function right below the function `get_total`? Now the code flow is not easy to follow, as caller statements and function definitions are mixed.

Reply:
Good idea!